test/.jshintrc

```diff
@@ -30,6 +30,6 @@
     },
     "predef":
         [
-            "describe", "beforeEach", "afterEach", "it"
+            "describe", "beforeEach", "afterEach", "it", "xdescribe", "xit"
```

Comment:
I have check there are 26 xdescribe() statement and 4 xit() statement added in this PR.

I understand that the work in sucesive PRs within the NGSI-LD line will be re-enable them, isn't it?

Reply:
Yes

Reply:
Ok, NTC